tests/common_tools.py

```diff
@@ -190,18 +190,25 @@ def create_valid_ticket_for_user_and_fare(user, fare=None):
     return ticket
 
 
-def get_default_conference():
+def get_default_conference(end=timezone.now() + timedelta(days=35)):
```

Comment:
The default `end` here will use the value of `timezone.now()` when the process is started (or module is loaded), not when the function is called, so it might result in unexpected behaviour. We could write it like:
```python
def get_default_conference(end=None):
    if not end:
        end = timezone.now() + timedelta(days=35)
    ...
```

Reply:
Good point, going to update this.